src/doc/intro.md

```diff
@@ -423,11 +423,11 @@ Let's see an example. This Rust code will not compile:
 use std::thread::Thread;
 
 fn main() {
-    let mut numbers = vec![1i, 2i, 3i];
+    let mut numbers = vec![1is, 2is, 3is];
```

Comment:
In this particular case code without type annotations would fail to compile (`error: unable to infer enough type information about`). However, to annotate only the first number would be sufficient. I could update the changed examples in such regard.